collector/lib/SysdigService.h

```diff
@@ -97,6 +98,7 @@ class SysdigService : public Sysdig {
   mutable std::mutex libsinsp_mutex_;
   std::unique_ptr<sinsp> inspector_;
   std::unique_ptr<sinsp_chisel> chisel_;
+  std::unique_ptr<ISignalServiceClient> grpc_client_;
```

Comment:
[nit] the `grpc_client_` naming here is a little imprecise, since we can have a `ISignalServiceClient` that does not talk to any gRPC server. Perhaps `signal_client_` or `signal_service_` or similar?